documentation/rdme.md

```diff
@@ -0,0 +1,206 @@
+
+With `rdme`, you can create workflows for a variety of use cases, including:
+
+- Syncing [OpenAPI/Swagger](https://docs.readme.com/docs/openapi) definitions (with support for bundling external references) 📦
+- Pre-upload validation (including OpenAPI 3.1) ✅
+- Syncing directories of Markdown files 📖
+
+## General Setup and Usage
+
+To see detailed CLI setup instructions and all available commands, check out [the `rdme` GitHub repository](https://github.com/readmeio/rdme#readme).
+
+### Markdown File Setup
+
+In order to sync a directory of Markdown files to your guides, you'll need to add certain attributes to the top of each page via a [YAML front matter block](https://jekyllrb.com/docs/front-matter/). See below for an example (using the page you're currently reading!):
+
+```markdown
+---
+title: CLI + GitHub Action Usage
+excerpt: Learn more about `rdme`, ReadMe's official CLI and GitHub Action!
+category: 5f7ce9e3a5504d0414d024c2
+---
+
+If you're anything like us...
+```
+
+The only required attributes are the `title` and `category`. To determine what your `category` value should be, you can use [the `Get all categories` endpoint](https://docs.readme.com/reference/getcategories) and grab the `id` value from the response.
+
+We automatically derive the page's slug via the file name (e.g. the file name `rdme.md` would become `/docs/rdme` in your ReadMe project), but you can override that as well:
+
+```markdown
+---
+title: CLI + GitHub Action Usage
+excerpt: Learn more about `rdme`, ReadMe's official CLI and GitHub Action!
+category: 5f7ce9e3a5504d0414d024c2
+slug: an-alternative-page-slug-example
+---
+```
+
+You can also specify several other page attributes in your YAML front matter, such as `hidden` (a boolean which denotes whether your page is published or unpublished). Any attributes you omit will remain unchanged on `rdme` runs. To view the full list of attributes, check out our [`Create doc` endpoint documentation](https://docs.readme.com/reference/createdoc).
+
+## GitHub Actions Usage
+
+[GitHub Actions](https://docs.github.com/actions) makes it easy to automatically execute workflows when certain events take place in your GitHub repository (e.g. new code is merged into the default branch, a new pull request is opened, etc.).
+
+<!-- TODO: it might be nice to have some sort of CI workflow that auto-updates the version in the examples below! -->
+<!-- TODO: populate marketplace listing link! -->
+
+> 📘 Keeping `rdme` up-to-date
+>
+> Note that the `@XX` in the below examples refers to the version of `rdme` (e.g. `@7.0`), which you can find in [the Marketplace listing](). We recommend [configuring Dependabot to keep your actions up-to-date](https://docs.github.com/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot).
```

Comment:
Marketplace listing link is missing. And should I publish a 7.0 release once this is merged?

Reply:
I have a TODO for that here: https://github.com/readmeio/rdme/blob/c4bd19851f14a45cb74bd9fe5c75c65dc2db5085/documentation/rdme.md#L62

I keep going back-and-forth on the version. How about we ship v6.5 once this is out, and then merge in #210 and immediately ship v7?

Reply:
sounds good to me